lib/rbac_service.rb

```diff
@@ -0,0 +1,48 @@
+require 'rbac-api-client'
+class RBACService
+  def self.call(klass)
+    setup
+    yield init(klass)
+  rescue RBACApiClient::ApiError => err
+    Rails.logger.error("RBACApiClient::ApiError #{err.message} ")
+    raise ServiceCatalog::RBACError, err.message
+  end
+
+  def self.paginate(obj, method, pagination_options, *method_args)
```

Comment:
If all pagination follows the same IPP it would seem that we could move this logic to the common-gem ( eventually )  or in a separate library within this service for reuse.

Reply:
@syncrou There is no consensus on this in the IPP yet. RBAC service is Django based which prefers page and page_size. So this implementation only works with that. We would need another one for limit and offset